formats/csv.js

```diff
@@ -1,16 +1,20 @@
 module.exports.type = "csv";
+var DEFAULT_DELIMITER = ','
 
-module.exports.is = function isCSV( sample ) {
-    return sample.toString().split( "\n" )[ 0 ].indexOf( "," ) != -1;
+module.exports.is = function isCSV( sample, delimiter ) {
```

Comment:
@alonbrody  The delimiter seems like one out of many possible options, which would make the arguments list too big. Instead, pass in an options object, that may have a `delimiter` key.